components/docs-chef-io/content/automate/ha_onprim_deployment_procedure.md

```diff
@@ -88,26 +88,24 @@ You can also view the [Sample Config](#sample-config).
 
 ## Steps to Deploy
 
-1. Continue with the deployment after generating the config:
+1. Continue with the deployment after generating the config. The deployment command will run the verify command internally
 
     ```bash
-    #Run commands as sudo.
-    sudo -- sh -c "
-    #Print data in the config
-    cat config.toml
-    #Run deploy command to deploy `automate.aib` with set `config.toml`
     chef-automate deploy config.toml --airgap-bundle automate.aib
-    "
+    ```
+
+   If we want to skip verficiation in the deployment command we can use `--skip-verify` flag
+    ```bash
+     chef-automate deploy config.toml --airgap-bundle automate.aib --skip-verify
     ```
 
 1. Verify the deployment by checking status summary
```

Comment:
Verify the deployment by checking status and info

Reply:
Why not the other 2 command

Reply:
Verify the deployment by running following commands